.github/pull_request_template.md

```diff
@@ -1,26 +1,12 @@
-# Objective
-<!-- Describe what this change achieves, and the details of how it works. -->
-
-# Motivation and Context
-<!--- Why is this change required? What problem does it solve? -->
-<!--- If it fixes an open issue, please link to the issue here. -->
-
-# Screenshots (if appropriate)
-
-# Checklist
-<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
-- [ ] If this contains visual changes, has it been reviewed by a designer?
-- [ ] If this introduces either a new component or a breaking change to an existing component, has it been reviewed by an advocate? (Ask in #prod_design_systems)
-- [ ] I have considered likely risks of these changes and got someone else to QA as appropriate
-- [ ] I have or will communicate these changes to the front end practice
-
-# Additional Considerations
-- Does there need to be right-to-left (RTL) options for localization and internationalization?
-- Has the test suite been updated?
-- Have you done cross-browser testing for our [supported browsers](https://academy.cultureamp.com/hc/en-us/articles/204539569-Supported-browsers-for-Participants)?
-- Have you updated any relevant documentation or left useful comments in the code?
-- Have you reviewed Culture Amp's Web Accessibility guide [Current Compliance, Going Forward, Statement and Answers for Customers](https://cultureamp.atlassian.net/wiki/spaces/Prod/pages/428572998/Web+Accessibility)?
-- If this contains substantial visual changes, especially far reaching ones, have you unlocked the Chromatic step in the pipeline?
-- Have Storybook stories been updated?
-
-**If you're new to Kaizen, please ask #prod_design_systems to set up an onboarding session to get you up to speed.** If you have an urgent PR to merge before that happens, it is safest to ask in #prod_design_systems to have a design system advocate review the PR to catch any issues.
+<!-- Hi there! 👋 -->
+<!-- If you are unsure about contributing to Kaizen, -->
+<!-- please reach out to us in #prod_design_systems on Slack. -->
+<!-- We are always happy to help you with any questions.  -->
+
+## Why
+<!-- Why have you created this PR? - Is it a new feature, or a bug fix? Or something else? -->
+<!-- Reference any relevant Jira tickets so your reviewer can find more context if needed. -->
```

Comment:
Probably worth adding—if there's a Github issue that the PR fixes, they can put `Fixes [issue URL]` to auto link and close the issue once this PR merges.

People forget to do it quite often, though we have moved away from Github issues a bit recently.

Reply:
Good reminder, I'll add it.